### PR TITLE
Update README to include QUICKSTART

### PR DESCRIPTION
**Pull request by bersace**
cc @daamien @julmon 

I would like to document docker & compose here. Since temboard README point to a separate file, i suggest to point it here.

This ease update of image and compose file documentation.

### CODE REVIEW

File: temboard/README.md [hunk 1]
```diff
@@ -1,36 +1,59 @@
...
+- a PG 9.5 cluster
+- a PG 9.6 cluster
+- a temboard agent for each PG cluster
+- a standard PG 9.6 cluster for the UI
+- a container for the UI
 
 After starting the docker-compose image, the UI is available on
+https://0.0.0.0:8888/ with `admin` / `admin` credentials.
+
+The agents can be accessed with `alice` / `alice` or `bob` // `bob`.
```
Comment (pgiraud) [inline temboard/README.md]: Extra slash

File: temboard/README.md [hunk 2]
```diff
@@ -1,36 +1,59 @@
...
+
+## Warnings
+
+**DO NOT USE THIS IN PRODUCTION**
+
+Obviously these docker images are designed for testing and demo. All database
+access are set to ``trust``, the SSL certificate is self-signed and the default
+passwords are public.
+
+Temboard agent is designed to run on same host of PostgreSQL which is
```
Comment (pgiraud) [inline temboard/README.md]: same host as

File: temboard/README.md [hunk 3]
```diff
@@ -1,36 +1,59 @@
...
+incompatible with Docker service-minded architecture. Temboard agent images
+requires access to docker socket to restart PostgreSQL. Which you do not want in
```
Comment (pgiraud) [inline temboard/README.md]: require

Comment (bersace) [inline temboard/README.md]: no, you never want to share docker socket to container in production :)

Comment (pgiraud) [inline temboard/README.md]: Sorry, I should be more explicit:
"Temboar agent images requires" -> "Temboard agent images require" (images is plural)

Comment (bersace) [inline temboard/README.md]: zut :(

**Pull request merged**